tests/yam/agama/agama.pm

```diff
@@ -24,6 +25,10 @@ sub run {
     assert_script_run("/usr/share/agama/system-tests/" . $test . ".cjs", timeout => 1200);
     script_run("dmesg --console-on");
 
+    Yam::Agama::agama_base::upload_agama_logs();
+    Yam::Agama::agama_base::upload_system_logs();
+    Yam::Agama::agama_base::upload_browser_automation_dumps();
```

Comment:
this is not present on success: https://openqa.opensuse.org/tests/4492091#step/agama/16,
you need to drop it, same for auto.